src/packages/tabbar/tabbar.harmony.css

```
@@ -0,0 +1,50 @@
+.nut-tabbar:last-child {
+  border-right: 0;
+}
+.nut-tabbar-fixed {
+  position: fixed;
+  bottom: 0px;
+  left: 0px;
+}
+.nut-tabbar-safe-area {
+  display: block;
+  width: 100%;
+  padding-bottom: constant(safe-area-inset-bottom);
```

Biome (1.9.4)

[error] Unexpected unknown function: constant

Use a known function instead.
See MDN web docs for more details.

(lint/correctness/noUnknownFunction)